CHANGES.rst

```diff
@@ -416,8 +420,8 @@ Unreleased
 * Return values from :meth:`.Comment.block`, :meth:`.Message.block`,
   :meth:`.SubredditMessage.block`, :meth:`.SubredditFlair.delete`,
   :meth:`.friend`, :meth:`.Redditor.message`, :meth:`.Subreddit.message`,
-  :meth:`.select`, and :meth:`.unfriend` are removed as they do not provide
-  any useful information.
+  :meth:`~.SubmissionFlair.select`, and :meth:`.unfriend` are removed as they
```

Comment:
If these methods are removed, does the link actually work?

Reply:
I don’t know, I should replicate it locally. But there’s really no reason for anyone to read that far back.